### ISSUE TITLE
Luminosity ratio is less than 4.5:1 for blocks lying on the toolbox on the page

### ISSUE THREAD

**Issue by Helen09** [microsoft/pxt-microbit#798]
**Repro steps:** 
• Navigate to https://makecode.microbit.org/beta
• Navigate to toolbox and open each toolbox 
• Verify the luminosity ratio for all the Text content

**Expect result**: 
Luminosity ratio should be more than 4.5:1 
**Actual result:** 
Luminosity ratio is less than 4.5:1 
![capture2](https://user-images.githubusercontent.com/23466737/40299846-186c3564-5cd7-11e8-8e3a-386946fa51b4.PNG)

**Comment by jwunderl** [comment]
tracking this at https://github.com/microsoft/pxt-microbit/issues/2271
